fontforge/Monocraft.sfd

```diff
@@ -10076,64 +10076,63 @@ Flags: W
 LayerCount: 2
 Fore
 SplineSet
-360 720 m 1,0,-1
- 360 840 l 1,1,-1
- 480 840 l 1,2,-1
- 480 720 l 1,3,-1
- 360 720 l 1,0,-1
-120 720 m 1,4,-1
- 120 840 l 1,5,-1
- 240 840 l 1,6,-1
- 240 720 l 1,7,-1
- 120 720 l 1,4,-1
-360 120 m 1,8,-1
- 480 120 l 1,9,-1
- 480 0 l 1,10,-1
- 360 0 l 1,11,-1
- 240 0 l 1,12,-1
- 120 0 l 1,13,-1
- 120 120 l 1,14,-1
- 240 120 l 1,15,-1
- 360 120 l 1,8,-1
-480 120 m 1,16,-1
- 480 240 l 1,17,-1
- 480 360 l 1,18,-1
- 480 480 l 1,19,-1
- 600 480 l 1,20,-1
- 600 360 l 1,21,-1
- 600 240 l 1,22,-1
- 600 120 l 1,23,-1
- 480 120 l 1,16,-1
-120 120 m 1,24,-1
- 0 120 l 1,25,-1
- 0 240 l 1,26,-1
- 0 360 l 1,27,-1
- 0 480 l 1,28,-1
- 120 480 l 1,29,-1
- 120 360 l 1,30,-1
- 120 240 l 1,31,-1
- 120 120 l 1,24,-1
-480 480 m 1,32,-1
- 360 480 l 1,33,-1
- 240 480 l 1,34,-1
- 120 480 l 1,35,-1
- 120 600 l 1,36,-1
- 240 600 l 1,37,-1
- 360 600 l 1,38,-1
- 480 600 l 1,39,-1
- 480 480 l 1,32,-1
-480 840 m 1,40,-1
- 480 960 l 1,41,-1
- 600 960 l 1,42,-1
- 600 840 l 1,43,-1
- 480 840 l 1,40,-1
-360 840 m 1,44,-1
- 240 840 l 1,45,-1
- 240 960 l 1,46,-1
- 360 960 l 1,47,-1
- 360 840 l 1,44,-1
+360 720 m 5,0,-1
+ 360 840 l 5,1,-1
+ 480 840 l 5,2,-1
+ 480 720 l 5,3,-1
+ 360 720 l 5,0,-1
+120 720 m 5,4,-1
+ 120 840 l 5,5,-1
+ 240 840 l 5,6,-1
+ 240 720 l 5,7,-1
+ 120 720 l 5,4,-1
+360 120 m 5,8,-1
+ 480 120 l 5,9,-1
+ 480 0 l 5,10,-1
+ 360 0 l 5,11,-1
+ 240 0 l 5,12,-1
+ 120 0 l 5,13,-1
+ 120 120 l 5,14,-1
+ 240 120 l 5,15,-1
+ 360 120 l 5,8,-1
+480 120 m 5,16,-1
+ 480 240 l 5,17,-1
+ 480 360 l 5,18,-1
+ 480 480 l 5,19,-1
+ 600 480 l 5,20,-1
+ 600 360 l 5,21,-1
+ 600 240 l 5,22,-1
+ 600 120 l 5,23,-1
+ 480 120 l 5,16,-1
+120 120 m 5,24,-1
+ 0 120 l 5,25,-1
+ 0 240 l 5,26,-1
+ 0 360 l 5,27,-1
+ 0 480 l 5,28,-1
+ 120 480 l 5,29,-1
+ 120 360 l 5,30,-1
+ 120 240 l 5,31,-1
+ 120 120 l 5,24,-1
+480 480 m 5,32,-1
+ 360 480 l 5,33,-1
+ 240 480 l 5,34,-1
+ 120 480 l 5,35,-1
+ 120 600 l 5,36,-1
+ 240 600 l 5,37,-1
+ 360 600 l 5,38,-1
+ 480 600 l 5,39,-1
+ 480 480 l 5,32,-1
+480 840 m 5,40,-1
+ 480 960 l 5,41,-1
+ 600 960 l 5,42,-1
+ 600 840 l 5,43,-1
+ 480 840 l 5,40,-1
+360 840 m 5,44,-1
+ 240 840 l 5,45,-1
+ 240 960 l 5,46,-1
+ 360 960 l 5,47,-1
+ 360 840 l 5,44,-1
```

Comment:
Everything looks good, though this character seems to have some metadata inadvertently changed. Let's undo that just in case it has an effect, though it likely doesn't.
```suggestion
360 720 m 1,0,-1
 360 840 l 1,1,-1
 480 840 l 1,2,-1
 480 720 l 1,3,-1
 360 720 l 1,0,-1
120 720 m 1,4,-1
 120 840 l 1,5,-1
 240 840 l 1,6,-1
 240 720 l 1,7,-1
 120 720 l 1,4,-1
360 120 m 1,8,-1
 480 120 l 1,9,-1
 480 0 l 1,10,-1
 360 0 l 1,11,-1
 240 0 l 1,12,-1
 120 0 l 1,13,-1
 120 120 l 1,14,-1
 240 120 l 1,15,-1
 360 120 l 1,8,-1
480 120 m 1,16,-1
 480 240 l 1,17,-1
 480 360 l 1,18,-1
 480 480 l 1,19,-1
 600 480 l 1,20,-1
 600 360 l 1,21,-1
 600 240 l 1,22,-1
 600 120 l 1,23,-1
 480 120 l 1,16,-1
120 120 m 1,24,-1
 0 120 l 1,25,-1
 0 240 l 1,26,-1
 0 360 l 1,27,-1
 0 480 l 1,28,-1
 120 480 l 1,29,-1
 120 360 l 1,30,-1
 120 240 l 1,31,-1
 120 120 l 1,24,-1
480 480 m 1,32,-1
 360 480 l 1,33,-1
 240 480 l 1,34,-1
 120 480 l 1,35,-1
 120 600 l 1,36,-1
 240 600 l 1,37,-1
 360 600 l 1,38,-1
 480 600 l 1,39,-1
 480 480 l 1,32,-1
480 840 m 1,40,-1
 480 960 l 1,41,-1
 600 960 l 1,42,-1
 600 840 l 1,43,-1
 480 840 l 1,40,-1
360 840 m 1,44,-1
 240 840 l 1,45,-1
 240 960 l 1,46,-1
 360 960 l 1,47,-1
 360 840 l 1,44,-1
```